### ISSUE TITLE
colorFills does not seem to be working on IOS safari and IOS chrome

### ISSUE THREAD

**Issue by Alex61NN5** [amcharts/amcharts3#190]
Hey,

this is my chart code 

```
this.AmCharts.makeChart('chartForecast', {
    'type': 'serial',
    'categoryField': 'category',
    'sequencedAnimation': false,
    'columnWidth': 0.3,
    'marginLeft': 19,
    'zoomOutButtonTabIndex': 1,
    'startDuration': 1,
    'accessible': false,
    'addClassNames': true,
    'autoResize': false,
    'categoryAxis': {
      'autoRotateCount': 0,
      'gridPosition': 'start',
      'autoGridCount': false,
      'axisAlpha': 0,
      'labelsEnabled': false,
      'showFirstLabel': false,
      'showLastLabel': false,
      'titleBold': false
    },
    'trendLines': [],
    'graphs': [
      {
        'balloonText': '[[title]] of [[category]]:[[value]]',
        'fillAlphas': 1,
        'id': 'AmGraph-1',
        'title': 'graph 1',
        'type': 'column',
        'lineAlpha': 0,
        'fillColors': ['#489f58', '#8ac249'],
        'valueField': 'column-1',
        'fixedColumnWidth': 20,
      },
      {
        'balloonText': '[[title]] of [[category]]:[[value]]',
        'fillAlphas': 1,
        'id': 'AmGraph-2',
        'title': 'graph 2',
        'type': 'column',
        'valueField': 'column-2',
        'fillColors': ['#f89d28', '#de602e'],
        'lineAlpha': 0,
        'lineThickness': 0,
        'fixedColumnWidth': 20,
      },
      {
        'bullet': 'round',
        'columnWidth': 0,
        'dashLength': 3,
        'fontSize': 1,
        'lineColor': '#FFFFFF',
        'id': 'AmGraph-3',
        'title': 'graph 3',
        'topRadius': 0,
        'valueField': 'column-1',
        'visibleInLegend': false
      },
      {
        'clustered': false,
        'columnWidth': 0,
        'fillAlphas': 1,
        'fillColors': ['#f89d28', '#de602e'],
        'fixedColumnWidth': 7,
        'id': 'AmGraph-4',
        'lineAlpha': 0,
        'lineThickness': 0,
        'showBalloon': false,
        'showOnAxis': true,
        'stackable': false,
        'tabIndex': 2,
        'title': 'graph 4',
        'topRadius': 0,
        'type': 'column',
        'valueField': 'column-4',
        'visibleInLegend': false
      },
      {
        'columnWidth': 0,
        'dashLength': 3,
        'fontSize': 1,
        'stackable': false,
        'lineColor': '#FFFFFF',
        'id': 'AmGraph-5',
        'title': 'graph 5',
        'topRadius': 0,
        'valueField': 'column-4',
        'visibleInLegend': false
      },
    ],
    'guides': [],
    'valueAxes': [
      {
        'id': 'ValueAxis-1',
        'stackType': 'regular',
        'autoGridCount': false,
        'autoRotateCount': 0,
        'axisAlpha': 0,
        'labelsEnabled': false,
        'showFirstLabel': false,
        'showLastLabel': false,
      }
    ],
    'allLabels': [],
    'balloon': {},
    'titles': [
      {
        'id': 'Title-1',
        'alpha': 0,
        'size': 15,
        'text': 'Chart Title'
      }
    ],
    'dataProvider': [
      {
        'category': 'category 1',
        'column-1': '6.5',
        'column-2': '-5.5',
        'column-3': '6.5',
        'column-4': '-2.0',
      },
      {
        'category': 'category 2',
        'column-1': '4.6',
        'column-2': '-6.6',
        'column-3': '4.6',
        'column-4': '-2.0',
      },
      {
        'category': 'category 3',
        'column-1': '7.8',
        'column-2': '-4.9',
        'column-3': '7.8',
        'column-4': '-2.0',
      }
    ]
});
```

Now I have a few columns and three of them are filled with a gradient using the property fillColors now it works perfectly on desktop but on mobile there's no gradient its just all black.. I cant find any official documentation on how to solve this issue or even any other references on similar issues.

Any Help would be appreciated

 I have a stackBlitz of this issue here https://stackblitz.com/edit/angular-fwuwet

**Comment by amcharts** [comment]
> New comment from Zendesk by **Anthony Piris** on ticket **38179**.
> (replying here will automatically notify amCharts support agent)

Hi there,

Your stackblitz isn't loading due to console errors but I'm not seeing the issue in my codepen here using your chart config: https://codepen.io/team/amcharts/live/aaca9fd9899341c9486dc388225e49f0

I just randomly tested on iOS 11 and 9 along with Safari 11 and 7 and the gradients are showing correctly for me. Chrome works as well.

Can you confirm you're seeing the issue on codepen as well? If so, can you provide what versions of mobile/desktop safari you're using? 

Best,

Anthony Piris
amCharts

**Comment by Alex61NN5** [comment]
Hmm.. Maybe it's an issue with the angular port of am-charts?? Because on my app its definitely not working. I'm not seeing the issue on your codepen

**Comment by amcharts** [comment]
> New comment from Zendesk by **Anthony Piris** on ticket **38179**.
> (replying here will automatically notify amCharts support agent)

Hi there,

I went and tweaked your stackblitz to make it load on my end and I'm still not seeing the problem: https://stackblitz.com/edit/angular-zb44uq

Tested on iOS 11, 10.2 and 9. and all the gradients are all showing correctly. I don't have access to iOS Chrome but it's the same rendering engine as Safari anyway.

Best,

Anthony Piris
amCharts
